README.md

```diff
@@ -42,18 +55,9 @@ the form:
 
     PANOPTICON_API_CREDENTIALS = { bearer_token: 'MY_BEARER_TOKEN' }
 
-
-## Middleware for request tracing
-
-We set a unique header at the cache level called `GOVUK-Request-Id`. In order
-to serve a user's request, if apps make API requests they should pass on this
-header, so that we can trace a request across the entire GOV.UK stack.
-
-`GdsApi::GovukRequestIdSniffer` middleware takes care of this. This gem contains
-a railtie that configures this middleware for Rails apps without extra effort.
-Other Rack-based apps should opt-in by adding this line to your `config.ru`:
-
-```use GdsApi::GovukRequestIdSniffer```
+In addition, the adapters can pass on a header containing the authenticated user
```

Comment:
I think it would be worth expanding on this a bit - it could be made clearer that these are two similar but separate forms of authorisation. i.e. the `bearer_token` method of authorisation is generally used to identify applications' API users to other endpoints, e.g. signon sends a bearer token for its "Signonotron API Client" user when it makes auth-related callbacks to apps to authenticate that it can make these requests. It may be that the opening paragraph above needs to be rewritten slightly to make the distinction between app-level identification with a bearer_token, and individual user identification with `HTTP_X_GOVUK_AUTHENTICATED_USER`.